home-page.html

```diff
@@ -11,11 +11,63 @@
     <link rel="preconnect" href="https://fonts.googleapis.com">
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
     <link href="https://fonts.googleapis.com/css2?family=Lato:wght@400;700;900&display=swap" rel="stylesheet">
+    <script src="script.js" defer></script>
+    <script src="contributors.js" defer></script>
+    <link rel="stylesheet" href="pop-up-menu.css">
 </head>
-<body>
+<body id="home-page">
     <section class="background-image-wrapper">
         <div class="over-lay">
-            <header class="about-mobile-head"><i class="fa-sharp fa-solid fa-bars"></i></header>
+            <header class="about-mobile-head"><i class="fa-sharp fa-solid fa-bars"></i>
+                <div id="pop-up-menu-wrapper">
+                    <i class="fa-solid fa-xmark" id="cancel"></i>
+                <div class="pop-up-menu">
+                    <ul class=".pop-up-list">
+                        <li class="pop-up-li"><a class="page-link" href="home-page.html">Home</a></li>
+                        <li class="pop-up-li"><a class="page-link" href="index.html">About</a></li>
+                        <li class="pop-up-li">Program</li>
```

Comment:
- [ ] Great job for adding the button in the mobile version  here but as I can see it is not found in desktop version in the given design but you forget to add it so please add and styleit to be the same  here to match the given design I will show you here 🙏🏻🔥🤩

- Your design
![image](https://user-images.githubusercontent.com/58553711/219921807-45ca7f3a-9303-4ec6-a93c-b437e59f082d.png)


- Given 
![image](https://user-images.githubusercontent.com/58553711/219342624-1ac76210-4790-4516-98fb-fc9f22ddf53f.png)